Makefile

```
@@ -112,12 +114,38 @@ endif
+	$(if $(filter $(1),arm64),\
+		CGO_ENABLED=1 CC=aarch64-linux-gnu-gcc CXX=aarch64-linux-gnu-g++ GOOS=linux GOARCH=$(1) LEDGER_ENABLED=$(LEDGER_ENABLED) \
+			go build -tags "$(build_tags)" -ldflags '$(ldflags)' $(if $(findstring nostrip,$(COSMOS_BUILD_OPTIONS)),,-trimpath) \
+			-o $(BUILDDIR)/$(1)/$(TREASURENET_BINARY) ./cmd/treasurenetd,\
+		GOOS=linux GOARCH=$(1) LEDGER_ENABLED=$(LEDGER_ENABLED) \
+			go build -tags "$(build_tags)" -ldflags '$(ldflags)' $(if $(findstring nostrip,$(COSMOS_BUILD_OPTIONS)),,-trimpath) \
+			-o $(BUILDDIR)/$(1)/$(TREASURENET_BINARY) ./cmd/treasurenetd)
+endef
+
+$(foreach arch,$(SUPPORTED_ARCHS),$(eval $(call ARCH_BUILD_TEMPLATE,$(arch))))
+
+build: build-$(BUILD_ARCH)
```

checkmake (0.2.2)

[warning] Target "build" should be declared PHONY.

(phonydeclared)